heareval/tasks/util/audio.py

```diff
@@ -69,3 +76,39 @@ def resample_wav(in_file: str, out_file: str, out_sr: int):
     )
     # Make sure the return code is 0 and the command was successful.
     assert ret == 0
+
+
+def audio_stats_wav(in_file: Union[str, Path]):
+    """Get statistics for a single wav file"""
+    audio = sf.SoundFile(in_file)
+    return {
+        "samples": len(audio),
+        "sample_rate": audio.samplerate,
+        "duration": round(len(audio) / audio.samplerate, 2),
```

Comment:
Why are we rounding here? Why not keep the precise value

Reply:
Just to keep it approximate and good in the json. But will remove it.